install/Knative-with-ICP.md

```diff
@@ -0,0 +1,193 @@
+    ```
+
+2. Create a cluster role for the pod security policy resource. The resourceNames for this role must be the name of the pod security policy that was created previous. Here we use `ibm-privileged-psp`. Run the following command:
+    ```shell
+    cat <<EOF | kubectl apply -f -
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: knative-role
+    rules:
+    -
+      apiGroups:
+        - extensions
+      resourceNames:
+        - ibm-privileged-psp
+      resources:
+        - podsecuritypolicies
+      verbs:
+        - use
+    EOF
+    ```
+
+3. Set up cluster role binding for the service account in Knative namespace. By using this role binding, you can set the service accounts in the namespace to use the pod security policy that you created.
+    ```shell
+    cat <<EOF | kubectl apply -f - 
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: knative-serving-psp-users
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: knative-role
+    subjects:
+    -
+      apiGroup: rbac.authorization.k8s.io
+      kind: Group
+      name: "system:serviceaccounts:knative-serving"
+    EOF
+    ```
+
+4. Use the same method to add `knative-build` and `knative-eventing` namespaces to the `ibm-privileged-psp` pod security policy.
+
+5. Ensure that the installation was successful by running the following commands until both of the Knative components show a `STATUS` of `Running`:
+
+    ```
+    kubectl get pods --namespace knative-serving
+    kubectl get pods --namespace knative-build
+    ```
+
+   It might take a few seconds for the Knative.
+
+    > Note: Instead of rerunning the command, you can add `--watch` to the above
+      command to view the component's status updates in real time. Use CTRL+C to exit watch mode.
+
+Now you can deploy an app to your newly created Knative cluster.
+
+## Deploying an app
+
+Now that your cluster has Knative installed, you're ready to deploy an app.
+
+If you'd like to follow a step-by-step guide for deploying your first app on
+Knative, check out the
+[Getting Started with Knative App Deployment](getting-started-knative-app.md)
+guide.
+
+If you'd like to view the available sample apps and deploy one of your choosing,
+head to the [sample apps](../serving/samples/README.md) repo.
+
+*Note*: When looking up the IP address to use for accessing your app, you need to look up
+  the NodePort for the `knative-ingressgateway` as well as the IP address used for ICP.
+  You can use the following command to look up the value to use for the {IP_ADDRESS} placeholder
+  used in the samples:
+  ```shell
+  echo $(ICP cluster ip):$(kubectl get svc knative-ingressgateway --namespace istio-system \
+  --output 'jsonpath={.spec.ports[?(@.port==80)].nodePort}')
+  ```
+
+## Cleaning up
+
+Delete the cluster on [IBM Cloud Private](https://www.ibm.com/cloud/private):
+
+```shell
```

Comment:
How about adding `shell` for other `curl` command?

Reply:
Done